app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -175,6 +174,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         setupThemeAndBrowsingMode(getModeFromIntentOrLastKnown(intent))
         setContentView(R.layout.activity_home)
 
+        navGraphJob = createNavGraphAsync()
```

Comment:
Can we comment why we're doing this and link to the associated bug? This will make it easier to re-evaluate this in the future.